src/winetricks

```diff
@@ -14105,6 +14105,37 @@ _EOF_
 
 #----------------------------------------------------------------
 
+w_metadata igorpro apps \
+title="Igor Pro 8" \
+publisher="WaveMetrics" \
+year="2018" \
+media="download" \
+file1="setupIgor8.exe" \
+installed_exe1="C:/Program Files/WaveMetrics/Igor Pro 8 Folder/IgorBinaries_Win32/Igor.exe"
+
+load_igorpro()
+{
+        if test "$W_ARCH" = "win64"; then
+            w_warn "Installation is recommended to be done on a 32bit wine prefix."
+        fi
+
+	w_set_winver win7
```

Comment:
Is there any reason for win7? Note that you can use `default` argument to use default wine environment (future-proofing)
https://github.com/Winetricks/winetricks/blob/1a3c72846a63c659b0b98073f201394ae2d08c36/src/winetricks#L2402